### DISCUSSION TITLE
kata 0

### DISCUSSION THREAD

**Discussion by PaoloHi** [LaunchX-InnovaccionVirtual/CursoIntroPython#289] [Q&A] [answered]
obtengo el mismo erro solamente en este modulo no entiendo por que 

import numpy as np
import matplotlib.pyplot as plt
data = np.random.default_rng(12345)
oxy_nums = data.integers(low=0, high=10, size=10)

plt.bar(range(len(oxy_nums)), oxy_nums)
plt.show()

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-4-f0034afe66fc> in <module>()
      1 import numpy as np
      2 import matplotlib.pyplot as plt
----> 3 data = np.random.default_rng(12345)
      4 oxy_nums = data.integers(low=0, high=10, size=10)
      5 

AttributeError: module 'numpy.random' has no attribute 'default_rng'

**Comment by JavierMendez-Coder** [comment, marked as answer]
Buenas noches. Al parecer el error que te esta apareciendo se puede dar por la versión de numpy que estas manejando (`<= 1.17`), con la siguiente linea de código dentro del notebook puedes revisar la versión del modulo:
```
print(np.__version__)
```
En caso de ser una versión antigua, puedes actualizar el modulo con esta línea de código en la consola/terminal/cmd:
```
pip install numpy --upgrade
```
Para referencia la versión actual de numpy es la `1.22.2`. Espero haberte ayudado. 👋
